bootstrap/config/config.go

```diff
@@ -62,6 +63,8 @@ const (
 // UpdatedStream defines the stream type that is notified by ListenForChanges when a configuration update is received.
 type UpdatedStream chan struct{}
 
+var defaultTimeout = 15 * time.Second
```

Comment:
should we make this as a constant somewhere so that you don't have to pass-in as an argument for several api calls.

Reply:
@brian-intel , I thought this got removed when moved to variables.go?